packages/website/src/shared/lib/utilities/enum.ts

```diff
@@ -1,5 +1,5 @@
 import _ from "lodash";
 
-export const getNthEnum = (enums: Record<any, any>, index: number) => _.chain(enums).values().nth(index).value();
+export const getNth = (enums: Record<any, any>, index: number) => _.chain(enums).values().nth(index).value();
 
```

Comment:
Maybe we can do something more vanillia?


```
function getNth(enums: Record<any, any>, index: number){
  const values = Object.values(enums)
  
  if(index.length > values.length) return undefined
  return values[index]
}

```

Reply:
Yeah, `lodash` is definitely just a convenience library. There's nothing it provies we can't do ourselves. Is there a reason why you rather not use it?